pkg/controllers/nodeclass/controller.go

```diff
@@ -84,18 +80,23 @@ func NewController(
 	capacityReservationProvider capacityreservation.Provider,
 	ec2api sdk.EC2API,
 ) *Controller {
-
+	reconcilers := []nodeClassReconciler{
```

Comment:
```suggestion
	reconcilers := []reconcile.TypedReconciler[*v1.NodeClass]{
```

Reply:
Previously resolved but the change wasn't made

Reply:
It should have been, I double checked and it is at least now.